app/controllers/declaration/ModelCacheable.scala

```diff
@@ -25,7 +25,12 @@ import scala.concurrent.{ExecutionContext, Future}
 trait ModelCacheable {
   val cacheService: ExportsCacheService
 
-  protected def updateHeaderLevelCache(
+  /*
+  Compilation warning
+    match may not be exhaustive.
+    [warn] It would fail on the following input: Left(_)
+   */
```

Comment:
Whats this for?

Reply:
I think Patryk added it as we are not covering the Left(_) case that could happen.
I'll add a case for the left scenario when we might not get a model returned from the cache.